.github/workflows/linkcheck.yml

```
@@ -0,0 +1,51 @@
+name: Link Check
+run-name: Link Check
+
+permissions: read-all
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    branches: ['main']
+
+  # run weekly on Monday at 8am UTC
+  schedule:
+    - cron: '0 8 * * 1'
+
+env:
+  CORE: ./core
+  DOCS: ./docs
+
+jobs:
+  linkcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: defenseunicorns/pepr
+          path: ./core
+          fetch-depth: 0
+
+      - uses: actions/checkout@v4
+        with:
+          repository: defenseunicorns/pepr-docs
+          path: ./docs
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Build the docs
+        run: '$DOCS/.github/workflows/scripts/build.sh'
```

Is this build process significantly different than how we build locally (with `npm build`)? In general, this seems like a case where the build behavior in CI should be a match for the build behavior locally since they use the same inputs.

```suggestion
      - name: Build the docs
        run: npm run build
```

---

We can't just `npm run build` because that can't generate the docs content. This is part of a bigger question/process I need to address. It is on my list to set up an easier way to build the docs locally for dev and testing.

---

Great, cross-reference an issue in the backlog here and we can consider this resolved